cEP-0005.md

```diff
@@ -49,9 +49,9 @@ paragraph:
   not a program and must remain flow insensitive.
 - To specify, what analysis should be run the user will not have to care about
   bears. Instead, an `aspects` setting will be provided allowing the user to
-  specify what aspects of their code should be analyzed. Aspects could be
-  `spelling` or `smells` and are namespaced such that `redundancy.clones` will
-  be possible as well. Aspects can be defined inclusive (analyze those aspects,
+  specify what aspects of their code should be analyzed. aspects could be
```

Comment:
@jayvdb Please please please let's not restart the discussion about the `aspects` spelling now :) This is counterproductive for GSoC ;) The **#aspectsTeam** has made the decision to keep it always-lower-case... And this is one of those little important fancy (maybe also funny, but anyway... even better) aspects that define the strong bonding inside the team

cc @coala/aspects-developers @sils @meetmangukiya @adtac